### ISSUE TITLE
Custom font size on project tree view

### ISSUE THREAD

**Issue by guobin211** [zed-industries/zed#4706]
### Check for existing issues

- [x] Completed

### Describe the feature

project tree font size too small

- [ ] support custom font size
- [ ] support custom line height

<img width="447" alt="image" src="https://github.com/zed-industries/community/assets/30451385/d4c39978-404e-4c91-a1e1-3ea3f42e4e08">

**Comment by shellscape** [comment]
I'd be curious to know if this can be modified via the new theming support that dropped this week in the preview build

**Comment by zawilliams** [comment]
The font size is indeed able to be updated now using, for example: `"ui_font_size": 18`.

**Comment by shellscape** [comment]
@zawilliams thanks for that tip! are there docs around this on the web yet?

**Comment by zawilliams** [comment]
@shellscape good question. Not on the docs site yet, but I found the info in a PR or an issue recently. You can see maybe a bit more in the default settings here: https://github.com/zed-industries/zed/blob/main/assets/settings/default.json#L46

**Comment by shellscape** [comment]
dope thanks. fwiw the default settings have pretty good comments in them (Settings -> Open Default). was just playing in there based on a suggestion in the docs.

**Comment by narqo** [comment]
I also wish there was `ui_line_height` setting to configure the line height of project tree. The default feels a bit too large, IMHO

**Comment by elliotlarson** [comment]
Yeah, I think it's important to be able to modify settings globally but also specifically.  I like the editor line height in general, but the line height in the file tree is too tall.  For this I'd prefer a line height closer to what VSCode/Cursor use.  It would be great to be able to set something like:

```json
"project_panel": {
  "ui_font_size": 13,
  "ui_line_height": 1.2
}
```

**Comment by ImSreyas** [comment]
> Yeah, I think it's important to be able to modify settings globally but also specifically. I like the editor line height in general, but the line height in the file tree is too tall. For this I'd prefer a line height closer to what VSCode/Cursor use. It would be great to be able to set something like:
> 
> "project_panel": {
>   "ui_font_size": 13,
>   "ui_line_height": 1.2
> }

Ya, i totally agree to this one.
